arcgis-ios-sdk-samples/Geometry/Cut geometry/CutGeometryViewController.swift

```diff
@@ -0,0 +1,156 @@
+            AGSPoint(x: -9882498.222673,  y: 6209888.368416, spatialReference: .webMercator()),
+            AGSPoint(x: -9821057.766387,  y: 6274562.532928, spatialReference: .webMercator()),
+            AGSPoint(x: -9690092.583250,  y: 6241417.023616, spatialReference: .webMercator()),
+            AGSPoint(x: -9605207.742329,  y: 6206654.660191, spatialReference: .webMercator()),
+            AGSPoint(x: -9564786.389509,  y: 6108834.986367, spatialReference: .webMercator()),
+            AGSPoint(x: -9449989.747500,  y: 6095091.726408, spatialReference: .webMercator()),
+            AGSPoint(x: -9462116.153346,  y: 6044160.821855, spatialReference: .webMercator()),
+            AGSPoint(x: -9417652.665244,  y: 5985145.646738, spatialReference: .webMercator()),
+            AGSPoint(x: -9438671.768711,  y: 5946341.148031, spatialReference: .webMercator()),
+            AGSPoint(x: -9398250.415891,  y: 5922088.336339, spatialReference: .webMercator()),
+            AGSPoint(x: -9419269.519357,  y: 5855797.317714, spatialReference: .webMercator()),
+            AGSPoint(x: -9467775.142741,  y: 5858222.598884, spatialReference: .webMercator()),
+            AGSPoint(x: -9462924.580403,  y: 5902686.086985, spatialReference: .webMercator()),
+            AGSPoint(x: -9598740.325877,  y: 5884092.264688, spatialReference: .webMercator()),
+            AGSPoint(x: -9643203.813979,  y: 5845287.765981, spatialReference: .webMercator()),
+            AGSPoint(x: -9739406.633691,  y: 5879241.702350, spatialReference: .webMercator()),
+            AGSPoint(x: -9783061.694736,  y: 5922896.763395, spatialReference: .webMercator()),
+            AGSPoint(x: -9844502.151022,  y: 5936640.023354, spatialReference: .webMercator()),
+            AGSPoint(x: -9773360.570059,  y: 6019099.583107, spatialReference: .webMercator()),
+            AGSPoint(x: -9883306.649729,  y: 5968977.105610, spatialReference: .webMercator()),
+            AGSPoint(x: -9957681.938918,  y: 5912387.211662, spatialReference: .webMercator()),
+            AGSPoint(x: -10055501.612742, y: 5871965.858842, spatialReference: .webMercator()),
+            AGSPoint(x: -10116942.069028, y: 5884092.264688, spatialReference: .webMercator()),
+            AGSPoint(x: -10111283.079633, y: 5933406.315128, spatialReference: .webMercator()),
+            AGSPoint(x: -10214761.742852, y: 5888134.399970, spatialReference: .webMercator()),
+            AGSPoint(x: -10254374.668616, y: 5901877.659929, spatialReference: .webMercator())
+            ]
+        
+        // create a polygon from the array of points
+        let polygon = AGSPolygon(points: points)
+        
+        // create a blue border line symbol with a stroke weight of 4 for the Lake Superior polygon
+        let lineSymbol = AGSSimpleLineSymbol(style: .solid, color: .blue, width: 4)
+        
+        // create a semi transparent blue
+        let fillColor = UIColor(red: 0, green: 0, blue: 1, alpha: 0.1)
```

Comment:
You might consider using a color literal (`#colorLiteral`) and combining this line into line `70`.